pwclient/api.py

```diff
@@ -581,6 +581,28 @@ def _detail(
         data, _ = self._get(url)
         return json.loads(data)
 
+    @staticmethod
+    def _get_next_page(headers):
+        link_header = next((data for header, data in headers if header == 'Link'), None)
+        if link_header is None:
+            return None
+
+        rel = '; rel="next"'
```

Comment:
According to [MDN's documentation about the Link header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Link) this is overspecific. There might be other parameters than `rel` and `next` might also be unquoted. I looked up a few implementations for parting the Link header and found several different ones. All in all this isn't trivial to implement if you also want it to be somewhat robust. My position here would be to better use the implementation from the requests package than to add another imperfect implementation.

Reply:
That was my original thought. However, it was pointed out that we really only have to deal with the link header implementation from patchwork itself.